code/dynamic_programming/src/longest_repeating_subsequence/longest_repeating_subsequence.cpp

```diff
@@ -0,0 +1,34 @@
+// Part of Cosmos by OpenGenus Foundation
+
+//dynamic programming || Longest repeating subsequence
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+int longestRepeatingSubsequence(std::string s)
+{
+    int n = s.size();  // Obtaining the length of the string
+    std::vector<std::vector<int> >dp( n+1, std::vector<int>(n+1, 0) );
```

Comment:
Remove space between greater-than sign.

Reply:
Please read coding style for space.
`dp( n+1, std::vector<int>(n+1, 0) )` >> `dp(n + 1, std::vector<int>(n + 1, 0))`